source/Audio.cpp

```diff
@@ -294,10 +350,6 @@ void Audio::PlayMusic(const string &name)
 	if(!isInitialized)
 		return;
 
-	// Skip changing music if the requested music is already playing.
-	if(name == currentTrack->GetSource())
```

Comment:
could you develop a bit why you're changing this?

Reply:
resolved